packages/odf/components/s3-browser/upload-objects/upload-status/UploadStatusItem.tsx

```diff
@@ -1,23 +1,46 @@
 import * as React from 'react';
+import { humanizeBinaryBytes } from '@odf/shared/utils';
+import { observer } from 'mobx-react-lite';
 import {
   Button,
   ButtonVariant,
   Flex,
   FlexItem,
   Icon,
-  Progress,
   ProgressMeasureLocation,
+  Progress,
+  ProgressVariant,
 } from '@patternfly/react-core';
 import { CloseIcon, FileIcon } from '@patternfly/react-icons';
+import { uploadStore } from '../store';
+import { UploadStatus } from '../types';
 import './uploadStatusItem.scss';
 
 type UploadStatusItemProps = {
   fileName: string;
   fileSize: string;
   progress: number;
-  onAbort?: () => void;
   failed: boolean;
-  variant: Progress['props']['variant'];
+  itemKey: string;
+};
+
+const getProgressVariant = (
+  state: UploadStatus,
+  // For cases whjen it's complete but user presses cancel(edge case)
+  isComplete: boolean
+): Progress['props']['variant'] => {
+  if (isComplete) {
+    return undefined;
```

Comment:
if upload is complete shouldn't we show success ??
```suggestion
    return ProgressVariant.success;
```

Reply:
makes sense. Thanks!